src/index.js

```diff
@@ -53,6 +54,8 @@ class Node extends EventEmitter {
       this.swarm.on('peer-mux-closed', (peerInfo) => {
         this.emit('peer:disconnect', peerInfo)
       })
+
+      this.modules.transport.push(new Circuit(this.swarm, _options.relay))
```

Comment:
This is not what we are looking for. Relay should not be added as a transport (although it uses the same interface). It should be activated in libp2p-swarm.